lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -198,7 +219,7 @@ def test_empty_query(
         ),
     )
     def test_http_exception(
-        self, server_config, query_api, exceptions, user_ok, find_template
+        self, server_config, query_api, exceptions, user_ok, find_template, pbench_token
```

Comment:
I don't think you need to add `pbench_token` here -- this test doesn't seem to reference it, and the fixtures that need it have it baked in.